wolfcrypt/test/test.c

```diff
@@ -32,6 +32,28 @@
 
 #ifndef NO_CRYPT_TEST
 
+#if defined(WOLFSSL_STATIC_MEMORY) && !defined(HAVE_IO_POOL)
+    #ifndef HAVE_PKCS7
```

Comment:
This ifndef is here because our tests in test.c currently do not read cert buffers with the PKCS7 test function.